websites/O/OK/metadata.json

```diff
@@ -0,0 +1,66 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "257151906038677504",
+		"name": "SHISEN"
+	},
+	"service": "OK",
+	"description": {
+		"en": "OK is a social network service used mainly in Russia and former Soviet Republics.",
+		"ru": "ОК — социальная сеть, используемая в основном в России и бывших советских республиках."
+	},
+	"url": "ok.ru",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Jky2SvM.png",
```

Comment:
Logos should be 512x512px at minimum. Find a larger version or upscale it using a tool like waifu2x.

Reply:
> Logos should be 512x512px at minimum. Find a larger version or upscale it using a tool like waifu2x.

Okay, I'll reupload.